### PR TITLE
sounds halfway named

### PR DESCRIPTION
**Pull request by HailSanta**

### CODE REVIEW

**Review by ethteck — APPROVED** [review]

File: include/enums.h [hunk 1]
```diff
@@ -549,119 +549,119 @@ enum SoundIDs {
...
+    SOUND_005A                                    = 0x0000005A,
+    SOUND_LRAW_005B                               = 0x0000005B,
+    SOUND_005C                                    = 0x0000005C,
+    SOUND_LRAW_005D                               = 0x0000005D,
+    SOUND_005E                                    = 0x0000005E,
+    SOUND_005F                                    = 0x0000005F,
+    SOUND_LRAW_0060                               = 0x00000060,
+    SOUND_0061                                    = 0x00000061,
+    SOUND_0062                                    = 0x00000062,
+    SOUND_0064                                    = 0x00000064,
```
Comment (nanaian) [inline include/enums.h:609]: Too much whitespace here?

File: include/enums.h [hunk 2]
```diff
@@ -701,37 +701,37 @@ enum SoundIDs {
...
+    SOUND_00F9                                    = 0x000000F9,
+    SOUND_00FA                                    = 0x000000FA,
+    SOUND_00FB                                    = 0x000000FB,
+    SOUND_00FC                                    = 0x000000FC,
+    SOUND_00FD                                    = 0x000000FD,
+    SOUND_00FE                                    = 0x000000FE,
+    SOUND_00FF                                    = 0x000000FF,
+    SOUND_0100                                    = 0x00000100,
+    SOUND_0108                                    = 0x00000108,
+    SOUND_010B                                    = 0x0000010B,
```
Comment (nanaian) [inline include/enums.h:721]: "

**Review by nanaian — CHANGES_REQUESTED** [review]
Looks like the `=` for unnamed sounds arent aligned

File: include/enums.h (hunk 1 above)
Comment (HailSanta) [reply include/enums.h:609]: I intentionally added two extra spaces for sounds yet-to-be-named so I can spot them more quickly.

**Pull request merged**